package.json

```diff
@@ -1,46 +1,75 @@
 {
-	"name": "springroll-container",
-	"version": "1.1.2",
-	"description": "The iframe controller for interacting with SpringRoll applications",
-	"main": "./dist/container.min.js",
-	"author":
-	{
-		"name": "Matt Karl",
-		"email": "matt@cloudkid.com"
-	},
-	"license": "MIT",
-	"repository":
-	{
-		"type": "git",
-		"url": "https://github.com/SpringRoll/SpringRollContainer"
-	},
-	"dependencies":
-	{
-		"bellhop-iframe": "~2.0.0"
-	},
-	"devDependencies":
-	{
-		"grunt": "~0.4.5",
-		"grunt-file-creator": "^0.1.3",
-		"library-grunt": "~0.6.0",
-		"grunt-jsbeautifier": "~0.2.10",
-		"grunt-contrib-copy": "^0.8.2"
-	},
-	"keywords": [
-		"deployment",
-		"game",
-		"springroll",
-		"iframe",
-		"runtime"
-	],
-	"scripts":
-	{
-		"test": "grunt default test --verbose"
-	},
-	"readmeFilename": "README.md",
-	"bugs":
-	{
-		"url": "https://github.com/SpringRoll/SpringRollContainer/issues"
-	},
-	"homepage": "https://github.com/SpringRoll/SpringRollContainer"
-}
\ No newline at end of file
+  "name": "springroll-container",
+  "version": "2.0.0",
+  "description": "The iframe controller for interacting with SpringRoll applications",
+  "main": "./dist/index.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SpringRoll/SpringRollContainer"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.1.0",
+    "@babel/plugin-external-helpers": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
+    "babel-loader": "^8.0.2",
+    "chai": "^4.1.2",
+    "dotenv": "^6.0.0",
+    "eslint": "^5.6.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-node": "^7.0.1",
+    "eslint-plugin-prettier": "^2.6.2",
+    "karma": "^3.0.0",
+    "karma-babel-preprocessor": "^8.0.0-beta.0",
+    "karma-browserstack-launcher": "^1.3.0",
+    "karma-chai": "^0.1.0",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-cli": "^1.0.1",
+    "karma-es6-shim": "^1.0.0",
+    "karma-firefox-launcher": "^1.1.0",
+    "karma-html": "^1.0.5",
+    "karma-mocha": "^1.3.0",
+    "karma-requirejs": "^1.1.0",
+    "karma-webpack": "^4.0.0-rc.2",
+    "mocha": "^5.2.0",
+    "platform": "^1.3.5",
+    "requirejs": "^2.3.6",
+    "rollup": "^0.66.0",
+    "rollup-plugin-babel": "^4.0.3",
+    "rollup-plugin-commonjs": "^9.1.8",
+    "rollup-plugin-eslint": "^5.0.0",
+    "rollup-plugin-node-resolve": "^3.4.0",
+    "rollup-plugin-prettier": "^0.4.0",
+    "rollup-plugin-replace": "^2.0.0",
+    "rollup-plugin-terser": "^3.0.0"
+  },
+  "dependencies": {
```

Comment:
Generally I'm of the opinion of writing it the newer way and dropping dependancies as supported platforms change, and writing them ourselves does not make much sense either when someone else has already done that work.

I did however find a much better, more supported collection of pollyfills that I'll replace this with so we will only have one dependancy instead of 6 =)